src/System.Threading.Tasks.Dataflow/tests/project.json

```diff
@@ -27,7 +29,6 @@
     "netstandard1.3": {}
   },
   "supports": {
-    "coreFx.Test.netcore50": {},
```

Comment:
This is kind of unfortunate. We should try to get the RemoteExecutor stuff to work with netcore50/UWP eventually, but it's fine to have disabled for now.